src/connectors/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { Web3Provider } from '@ethersproject/providers'
+import { InjectedConnector } from '@web3-react/injected-connector'
+import { WalletConnectConnector } from '@web3-react/walletconnect-connector'
+import { WalletLinkConnector } from '@web3-react/walletlink-connector'
+import { PortisConnector } from '@web3-react/portis-connector'
+
+import { FortmaticConnector } from './Formatic'
+import { NetworkConnector } from './NetworkConnector'
+
+const NETWORK_URL = process.env.REACT_APP_NETWORK_URL
+const FORMATIC_KEY = process.env.REACT_APP_FORTMATIC_KEY
+const PORTIS_ID = process.env.REACT_APP_PORTIS_ID
```

Comment:
these need to start with NEXT_PUBLIC_ to work in the browser

Reply:
those consts weren't being used so just set them to nothing for right now. Once I set up coinbase, formatic, and portis I'll come back to this